react-basic-practice/src/components/Input/index.tsx

```diff
@@ -1,42 +1,15 @@
 import React from "react";
-import { Input as ChakraInput, InputProps } from "@chakra-ui/react";
+import { Input as ChakraInput, InputProps, useStyleConfig } from "@chakra-ui/react";
```

Comment:
Pls remove redundant code. Verify for all

Reply:
i done remove it and check all